tests/static/js/formatters-internal/image.js

```diff
@@ -170,14 +148,14 @@ describe('image formatter', () => {
       expect(euFileImgUrl).toEqual('https://dyn.eu.mktgcdn.com/f/0/FOO.jpg/fit=contain,width=200');
     });
 
-    it('do not transform image regardless of atLeastAsLarge\'s', () => {
+    it('do not transform image regardless of atLeastAsLarge', () => {
       let photoImgUrl = Formatters.image(photoImg, 'x', false).url;
       expect(photoImgUrl).toEqual('https://dyn.mktgcdn.com/p/FOO/fit=contain');
-      let photoWithPaddingImgUrl = Formatters.image(photoWithPaddingImg, 'x', false).url;
+      let photoWithPaddingImgUrl = Formatters.image(photoWithPaddingImg, '0x0', false).url;
```

Comment:
can we also update one of these tests to check `''`?

Reply:
actually, just saw your previous response to this. what do you mean by it being "forbidden from the caller"?

Reply:
Like I commented on the other thread, '' would throw an error. Added a test to cover that